.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,192 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - master
+    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
Do we really need `ready_for_review`/`reopened` here?

Reply:
Fixed.